### PR TITLE
evm: add forge deployment script

### PR DESCRIPTION
**Pull request by gator-boi**

### CODE REVIEW

File: evm/script/ConfigureWormholeNtt.s.sol
Comment (djb15) [inline evm/script/ConfigureWormholeNtt.s.sol:109]: What about Solana?

File: evm/script/ConfigureWormholeNtt.s.sol
Comment (djb15) [inline evm/script/ConfigureWormholeNtt.s.sol:128]: So by default every other chain will have the same inbound limit coming from a given chain?

**Review by djb15 — COMMENTED** [review]
This is cool!

Will approve once the rate limit skipping PR has landed as the script will have to change slightly to accept the extra constructor

File: evm/script/ConfigureWormholeNtt.s.sol
Comment (gator-boi) [reply evm/script/ConfigureWormholeNtt.s.sol:109]: Happy to add Solana before we merge this.

File: evm/script/ConfigureWormholeNtt.s.sol
Comment (gator-boi) [reply evm/script/ConfigureWormholeNtt.s.sol:128]: This script is executed once for each chain, so the integrator could modify the inbound limit for each chain that it's registering.

File: evm/script/DeployWormholeNtt.s.sol
Comment (djb15) [inline evm/script/DeployWormholeNtt.s.sol:85]: Could skip this is the rate limiter is disabled

File: evm/script/ConfigureWormholeNtt.s.sol
Comment (djb15) [inline evm/script/ConfigureWormholeNtt.s.sol:80]: Same, could skip this if rate limiting is disabled

**Review by solanoepalacio — DISMISSED** [review]

File: evm/README.md
Comment (RahulMaganti47) [inline evm/README.md:205]: add a note here to create the `testnet` dir before running this command

File: evm/README.md
Comment (RahulMaganti47) [inline evm/README.md:212]: add note to navigate back to the `evm` dir

**Review by RahulMaganti47 — COMMENTED** [review]
some readme additions

File: evm/README.md
Comment (gator-boi) [reply evm/README.md:205]: Whoops, I meant to create that ahead of time actually.

Comment (kcsongor) [reply evm/README.md:205]: better, include a mkdir command just before the cp step

Comment (gator-boi) [reply evm/README.md:205]: Sweet, that works.

File: evm/script/DeployWormholeNtt.s.sol
Comment (RahulMaganti47) [inline evm/script/DeployWormholeNtt.s.sol:49]: log address in 32-byte wormhole format

File: evm/script/DeployWormholeNtt.s.sol
Comment (RahulMaganti47) [inline evm/script/DeployWormholeNtt.s.sol]: same as above - log in 32-byte format

File: evm/script/DeployWormholeNtt.s.sol
Comment (RahulMaganti47) [inline evm/script/DeployWormholeNtt.s.sol:85]: log in 32-byte format

Comment (gator-boi) [reply evm/script/DeployWormholeNtt.s.sol:88]: fixed

**Review by RahulMaganti47 — APPROVED** [review]

**Review by nik-suri — APPROVED** [review]

**Pull request merged**
